src/adapters/MeetingsJSONAdapter.js

```diff
@@ -92,13 +92,27 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       observer.complete();
     });
 
-    // Attach local media stream if meeting `localVideo` property is not `null`.
+    // Attach media stream if meeting any of media stream property is not `null`.
     // We can not attach the MediaStream object in a JSON module, the work needs
     // to be done here.
-    const getMeetingWithLocalMedia$ = getMeeting$.pipe(
+    const getMeetingWithMedias$ = getMeeting$.pipe(
+      /* eslint-disable no-confusing-arrow */
```

Comment:
help me! If I put the whole expression inside the parentheses, `prettier` will remove them and then linter will complain. so I'm stuck inside this loop between linter and prettier.

Reply:
Can you disable the `prettier` run when you save?

Reply:
Prettier is configured with eslint. it will still not be validated.

Reply:
but it doesn't run automatically? at least it doesn't for me

Reply:
`npm run test:eslint` will error out.

Reply:
I'm good with changing the setting of no-confusing-arrow to match the new default: 
https://eslint.org/docs/6.0.0/user-guide/migrating-to-6.0.0#no-confusing-arrow-updates

Seems there's lots of discussion about it with prettier:
https://github.com/prettier/eslint-config-prettier/issues/90
https://github.com/prettier/eslint-config-prettier/issues/12
https://github.com/prettier/prettier/pull/1182

Reply:
your suggestion dint seem to work @adamweeks. as discussed in the space I will leave it disabled for now.